docs/maui/views/RatingView.md

```diff
@@ -0,0 +1,519 @@
+| Width | `double` | Gets the current width of this element. This is a read-only bindable property. |
+| WidthRequest | `double` | Gets or sets the desired width override of this element. This is a bindable property. |
+| Window | `Window` | Gets the Window that is associated with an element. This is a read-only bindable property. |
+| X | `double` | Gets the current X position of this element. This is a read-only bindable property. |
+| Y | `double` | Gets the current Y position of this element. This is a read-only bindable property. |
+| ZIndex | `int` | Gets or sets the front-to-back z-index of an element within a layout. This is a bindable property. |
+
+
+## Set custom shape
+The `CustomShape` property is a `string` that allows for the defining of custom rating item shape `path`. This feature empowers developers to implement unique designs, such as distinctive symbols, as rating items.
+
+Only when the `Shape` property is set to `Custom`, will the custom shape path be used.
+
+The following examples sets the custom and shape properties:
+
+```xaml
+<toolkit:RatingView
+	CustomShape="M 12 0C5.388 0 0 5.388 0 12s5.388 12 12 12 12-5.38 12-12c0-6.612-5.38-12-12-12z"
+	Shape="Custom" />
+```
+
+The equivalent C# code is:
+
+```csharp
+RatingView ratingView = new()
+{
+	CustomShape = "M 12 0C5.388 0 0 5.388 0 12s5.388 12 12 12 12-5.38 12-12c0-6.612-5.38-12-12-12z",
+	Shape = RatingViewShape.Custom,
+};
+```
+
+For more information about custom shapes, see [Shapes.Path](/dotnet/api/microsoft.maui.controls.shapes.path).
+
+
+## Set empty color
+The `EmptyColor` property is a `color` that for the unfilled (empty) rating items. This allows for clear visual differentiation between rated and unrated items.
+
+The following examples set the empty color property:
+
+```xaml
+<toolkit:RatingView
+	EmptyColor="Grey" />
+```
+
+The equivalent C# code is:
+
+```csharp
+RatingView ratingView = new()
+{
+	EmptyColor = Colors.Grey,
+};
+```
+
+## Set filled (rated) color
+The `FilleColor` property is a `color` that will be applied to the filled (rated) portion of each item, offering flexibility in defining the visual aesthetic of the rating items when selected by the user.
+
+The following examples set the filled color property:
+
+```xaml
+<toolkit:RatingView
+	FilledColor="Green" />
+```
+
+The equivalent C# code is:
+
+```csharp
+RatingView ratingView = new()
+{
+	FilledColor = Colors.Green,
+};
+```
+
+## Set is read only
+The `IsReadOnly` property is a `bool` that will set the control user interface to read only.
+
+The following examples set the is read only property:
+
+```xaml
+<toolkit:RatingView
+	IsReadOnly="False" />
+<toolkit:RatingView
+	IsReadOnly="True" />
+```
+
+The equivalent C# code is:
+
+```csharp
+RatingView editableRatingView = new()
+{
+	IsReadOnly = False,
+};
+RatingView readOnlyRatingView = new()
+{
+	IsReadOnly = True,
+};
+```
```

Comment:
Perhaps we just show how to make the control read-only and mention that this defaults to false?